### DISCUSSION TITLE
vue-radio/dist

### DISCUSSION THREAD

**Discussion by AST-esp-rasp** [pschatzmann/esp32_radio#39] [Q&A] [answered]
Hello,
clearly not an issue more a question to understand how it works.
from what i can read the server provides a webpage and the content of this page is loaded from  https://github.com/pschatzmann/esp32_radio/tree/master/vue-radio/dist 
?
I'm asking because I would like to change the web interface.

**Comment by pschatzmann** [comment, marked as answer]
I think this is explained in https://github.com/pschatzmann/esp32_vue_example
